src/wallet/rpcdump.cpp

```diff
@@ -686,6 +687,8 @@ UniValue z_importviewingkey(const UniValue& params, bool fHelp)
             + HelpExampleCli("z_importviewingkey", "\"vkey\" whenkeyisnew 30000") +
```

Comment:
Nitpick: the line 684 above
```  + HelpExampleCli("z_importviewingkey", "\"vkey\", no") +``` has a stray comma

Reply:
Fixed